src/binary-reader.cc

```diff
@@ -2220,7 +2220,7 @@ Result BinaryReader::ReadElemSection(Offset section_size) {
     }
     Type elem_type = Type::Funcref;
 
-    CALLBACK(BeginElemSegment, i, table_index, flags, elem_type);
+    CALLBACK(BeginElemSegment, i, table_index, flags);
```

Comment:
So was this simple report the Funcref in all cases before?  It looks like it.  I think I remember having a mental TODO about this.

Reply:
Exactly, the elem_type was changed below, but the callback always would pass Funcref.